pkg/controller/disruption/disruption.go

```diff
@@ -470,9 +470,15 @@ func (dc *DisruptionController) Run(ctx context.Context) {
 		return
 	}
 
-	go wait.UntilWithContext(ctx, dc.worker, time.Second)
+	for i := 0; i < pdbWorkers; i++ {
+		go wait.UntilWithContext(ctx, dc.worker, time.Second)
+	}
+
 	go wait.Until(dc.recheckWorker, time.Second, ctx.Done())
-	go wait.UntilWithContext(ctx, dc.stalePodDisruptionWorker, time.Second)
+
+	for i := 0; i < stalePodWorkers; i++ {
```

Comment:
Is it useful to increase the number of workers for both the pdb processing and stale conditions?

Reply:
+1, can you break down the perf implications of each option? @xigang

Reply:
I don't have specific performance metrics yet, but I was concerned that a single worker might limit throughput, so I increased the number of concurrent workers.

Reply:
in addition, have we checked that the reconcile loop can be parallelized?

Reply:
@aojea Yes, the `stalePodDisruptionWorker` reconciliation loop is safe for parallelization. It uses thread-safe workqueues and processes pod keys independently. Each worker safely updates the Pod's DisruptionTarget condition to False when it becomes stale (after `stalePodDisruptionTimeout`), using the k8s API's UpdateStatus endpoint.